src/pages/question/Form/steps/FeaturesFromStep/FeaturesFormStep.tsx

```diff
@@ -178,7 +178,15 @@ export const FeaturesFormStep: FC = () => {
         </div>
 
         <div className="flex flex-col mt-4">
-          <h2>Conhecimento a Ser Verificado</h2>
+          <h2>Intenção</h2>
+          <input
+            className="block rounded p-1 w-full border-gray-400 border shadow-sm"
```

Comment:
você precisa passar o valor default aqui para na ele puxar na edição, acabei esquecendo de fazer o mesmo com o campo targedKnowledge...
para passar a pedir esses dados na query é só subir no início desse arquivo e adicionar no FeatureFragments. Assim todo lugar que usa o componente featureFormStep vai saber exatamente o que precisa buscar para montar o componente.
```
targetedKnowledge
intention
```